languages/python/polar/query.py

```diff
@@ -60,17 +56,17 @@ def run(self, ffi_query=None):
             if kind == "MakeExternal":
                 self.handle_make_external(data)
             if kind == "ExternalCall":
-                self.handle_external_call(ffi_query, data)
+                self.handle_external_call(self.ffi_query, data)
```

Comment:
This doesn't have to happen on this PR, but we can just access ``self.ffi_query`` from within the handler methods.

Reply:
fixed.